xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/wizard/wizard.css

```diff
@@ -116,7 +116,6 @@
 }
 
 .wizard-body .half{
-  width: 48%;
```

Comment:
It's because I have [XWIKI-12560](http://jira.xwiki.org/browse/XWIKI-12560) in mind, so I'm surprise it works in master.